docs/about/changelog.md

```diff
@@ -3,6 +3,11 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## 3.0.1
```

Comment:
Ugh?? Sup Git? :/

Reply:
Yup. this file is copy/paste. Not sure WHY this is needed it as we could just provide the link to changelog on main repo, but it was not my decision at the time, just like the idea with hugo.
Also this file has weirdly a bit different formatting. ### vs ##

Reply:
Will remove.